modules.d/95iscsi/module-setup.sh

```diff
@@ -253,20 +253,13 @@ install() {
                       $systemdsystemunitdir/iscsiuio.socket \
                       iscsiadm iscsid
 
-        mkdir -p "${initdir}/$systemdsystemunitdir/sockets.target.wants"
         for i in \
                 iscsid.socket \
                 iscsiuio.socket \
-            ; do
-            ln_r "$systemdsystemunitdir/${i}" "$systemdsystemunitdir/sockets.target.wants/${i}"
-        done
-
-        mkdir -p "${initdir}/$systemdsystemunitdir/basic.target.wants"
-        for i in \
                 iscsid.service \
                 iscsiuio.service \
             ; do
-            ln_r "$systemdsystemunitdir/${i}" "$systemdsystemunitdir/basic.target.wants/${i}"
+            systemctl --root "$initdir" enable "$i"
```

Comment:
iscsid.service has:
```
[Install]
WantedBy=multi-user.target
```
Which is not the `basic.target`. This is why I didn't use systemctl in the first place. There is no `multi-user.target` in the initramfs.

Reply:
Thank you. I didn't realize it.